content/en/tracing/guide/metrics_namespace.md

```diff
@@ -20,6 +20,7 @@ further_reading:
 aliases:
     - /tracing/getting_further/metrics_namespace
 ---
+<div class="alert alert-warning">This method of using trace metrics is outdated. Instead, [tracing distribution metrics using DDSketch][11] is recommended.
```

Comment:
```suggestion
<div class="alert alert-warning">This method of using trace metrics is outdated. Instead, <a href="/tracing/guide/ddsketch_trace_metrics/">tracing distribution metrics using DDSketch</a> is recommended.</div>
```

Reply:
oh you're totally right! I was trying to be fast and fix things without my brain